Utilities/Hook.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace OnboardingSpecflowProject.Utilities
+{
+    internal class Hook :CommonDriver
+    {
+        //public static IWebDriver driver;
+        [Before]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
```

Comment:
this should be done in the Common driver, here you should create an instance of Common driver and then call the  initialise()


public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
done extend the Hooks to ->  :CommonDriver